test/cmr/sizing/tests/unit/formats.clj

```diff
@@ -0,0 +1,33 @@
+(ns cmr.sizing.tests.unit.formats
```

Comment:
Good use of unit tests.

You will also need to update the service-bridge integration tests with the new ASCII estimates.

Reply:
This is the part I don't know what to do. So it's a different repo, so I will have to submit a separate PR? Also, since the tests are run against sit, I won't be able to verify until after it's merged?